script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -388,17 +387,17 @@ Even a question of whether a life exists or not is irrelevant. If it moves and c
 [f5a3c6162601239e485964eb636ec553553f92a6]{
 -- Page 302, Offset 17316.
 -- 死という負債を、他人の血で誤魔化し続ける吸血鬼だから」
-It is a vampire that continues to bypass the debt of death with the blood of others."
+It's a vampire that continues to cheat the debt called death with the blood of others."
```

Comment:
It's a vampire that continues to cheat that debt we call death with the blood of others.